common/src/main/java/eu/europeana/fulltext/util/NormalPlayTime.java

```diff
@@ -0,0 +1,207 @@
+package eu.europeana.fulltext.util;
+
+import java.text.ParseException;
+
+/**
+ * Created by luthien on 2019-06-13, based on gdata-java-client
+ * Time specification object which tries to conform to section 3.6
+ * of RFC 2326 (Normal Play Time).
+ * <p>
+ * It only supports a millisecond precision. Any time more precise than
+ * that will be lost when parsing.
+ */
+public class NormalPlayTime {
+
+    /**
+     * The NormalPlayType 'now'
+     */
+    public static final NormalPlayTime NOW = new NormalPlayTime(true, -1L);
+
+    private boolean isNow;
+    private long    ms;
+
+    /**
+     * Creates a NormalPlayTime object.
+     * @param ms time offset, in milliseconds
+     */
+    public NormalPlayTime(long ms) {
+        this(false, ms);
+    }
+
+    private NormalPlayTime(boolean now, long ms) {
+        isNow = now;
+        this.ms = ms;
+    }
+
+    /**
+     * Check whether this time is the special 'now' time.
+     */
+    public boolean isNow() {
```

Comment:
I don't understand the concept 'now' I think. What does this do? Do we use it?

Reply:
I created this class based on [gdata-java-client ](https://github.com/google/gdata-java-client/blob/c6202a55f5f29afb37ffcf876674dca372f3fb4c/java/src/com/google/gdata/data/media/mediarss/NormalPlayTime.java); I suppose that it had some function in the original context.

I took it out now.